web/public/locales/es.json

```
@@ -74,7 +74,11 @@
     "now": "Ahora"
   },
   "time-controller": {
-    "title": "Mostrar datos del pasado"
+    "title": "Mostrar datos del pasado",
+    "daily": "diario",
+    "hourly": "cada hora",
```

```suggestion
    "hourly": "horario",
```
I think this is better

---

I agree with @silimotion :) In Spanish we say "diario" and "horario"